bin/start_sync_stop.py

```diff
@@ -119,6 +127,7 @@ async def rippled(exe=DEFAULT_EXE, config_file=DEFAULT_CONFIGURATION_FILE):
 
         code = await process.wait()
         end = time.time()
+        logfile.close()
         logging.info(
             f'rippled stopped after {end - start:.1f} seconds with code {code}'
         )
```

Comment:
I have a tangential question (unrelated to this PR) about the `loop()` function in this file.

How is the for-loop on line 185 terminating? I understand that we are dumping the logs in uniquely named files, but how will that function ever stop? Is it supposed to be manually terminated through Ctrl+C (triggering the `KeyboardInterrupt` on line 253) ?

Reply:
It is meant to be an infinite loop terminated by keyboard interrupt.